Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py

```diff
@@ -252,34 +261,38 @@ def get_ip_indicator_reputation(stix_object: dict, reputation_score: int, reliab
         integration_name=INTEGRATION_NAME,
         score=reputation_score,
         reliability=reliability_score,
+        message='No results found.' if is_unknown else None
     )
 
-    ip_addr = extract_indicator_from_pattern(stix_object["pattern"])
+    indicator_value: str = extract_indicator_from_pattern(stix_object)
+
     ip = Common.IP(
-        ip=ip_addr,
+        ip=indicator_value,
         dbot_score=dbot_score,
         traffic_light_protocol=tlp,
     )
     return CommandResults(
         outputs_prefix="SEKOIAIntelligenceCenter.IP",
         outputs_key_field="name",
-        outputs=stix_object,
+        outputs=None if is_unknown else stix_object,
         indicator=ip,
     )
 
 
-def get_file_indicator_reputation(stix_object: dict, reputation_score: int, reliability_score: str, tlp: str) -> CommandResults:
+def get_file_indicator_reputation(stix_object: dict, reputation_score: int, reliability_score: str,
+                                  tlp: str, is_unknown: bool) -> CommandResults:
     """
     Return stix_object of type file as indicator
     """
 
-    hashes = extract_file_indicator_hashes(stix_object["pattern"])
+    hashes = extract_file_indicator_hashes(stix_object.get("pattern"))
```

Comment:
You don't return any indicator when you don't have pattern, so you are returning empty hash dict. As a result, we do not have a verdict (I saw it in the last demo). we need to return some value here.